libs/core/algorithms/include/hpx/parallel/container_algorithms/uninitialized_default_construct.hpp

```diff
@@ -15,7 +15,7 @@ namespace hpx { namespace ranges {
     // clang-format off
 
     /// Constructs objects of type typename iterator_traits<ForwardIt>
-    /// ::value_type in the uninitialized storage designated by the range
+    /// in the uninitialized storage designated by the range
```

Comment:
Let's not remove the `>::value_type` part, rather have the whole `typename iterator_traits<ForwardIt>::value_type` on its own line.